cmd/start.go

```diff
@@ -183,6 +187,35 @@ func Start(c *cli.Context) error {
 	}
 }
 
+func initializeExtensions(
+	ctx context.Context,
+	config config.Extensions,
+	ethereumChain *ethereum.EthereumChain,
+) {
+	if len(config.TBTC.TBTCSystem) > 0 {
```

Comment:
If `[Extensions.TBTC]` is not defined in the config file, is `config.TBTC` not `nil`? I wonder if we perhaps can reduce it to `config.TBTC != nil`.

Reply:
It's a struct, not a pointer so it won't be `nil`.